packages/manager/apps/web-office/public/translations/dashboard/users/order-licenses/Messages_fr_FR.json

```
@@ -0,0 +1,6 @@
+{
+  "dashboard_users_order_licences_quantity": "Nombre :",
+  "dashboard_users_order_licences_type": "Type de licence :",
```

```suggestion
  "dashboard_users_order_licences_quantity": "Nombre",
  "dashboard_users_order_licences_type": "Type de licence",
```

if it's label no need ":"